control-plane/subcommand/server-acl-init/command_test.go

```diff
@@ -1284,6 +1285,8 @@ func TestRun_NoLeader(t *testing.T) {
 			numACLBootCalls++
 		case "/v1/agent/self":
 			fmt.Fprintln(w, `{"Config": {"Datacenter": "dc1", "PrimaryDatacenter": "dc1"}}`)
+		case "/v1/acl/tokens":
+			fmt.Fprintln(w, `[]`)
```

Comment:
Some of these tests get modified because now we do a token list call to ensure we're not recreating server tokens.